### DISCUSSION TITLE
How to integrate this with my custom code?

### DISCUSSION THREAD

**Discussion by namanbhardwaj46** [QIN2DIM/hcaptcha-challenger#303] [Q&A]
This is great hcaptcha solver, i am looking for this kind of solution, but how can i integrate this with my custom code, so it can solve captcha on different urls? Any help would be appreciated.

Thankyou contributors
Great Work

**Comment by QIN2DIM** [comment]
If you are using python, try looking at [pypi docs issues](https://github.com/QIN2DIM/hcaptcha-challenger/issues/226) for a breakthrough. If it's another application scenario, you can provide more requirements.
